frontend/benefit/applicant/src/components/applications/applicationList/useApplicationList.ts

```diff
@@ -93,8 +92,11 @@ const useApplicationList = (status: string[]): ApplicationListProps => {
       default:
         return {
           label: t(`${translationListBase}.common.check`),
-          // implement the action
-          handleAction: noop,
+          handleAction: (): void => {
+            void router.push(
+              `${ROUTES.APPLICATION_FORM}?id=${id}&isReadOnly=true`
```

Comment:
what if user just decides to remove the `&isReadOnly=true` param could it be a harmful thing?